app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require('./errors')
+const { getTopics } = require('./controllers/topics-controller')
+
+const app = express()
```

Comment:
As were not taking any json from the user this isn't currently necessary this is just something to be aware of

Reply:
app.use(express.json())